eclass/linux-mod-r1.eclass

```diff
@@ -545,10 +557,21 @@ modules_post_process() {
 	(( ${#mods[@]} )) ||
 		die "${FUNCNAME[0]} was called with no installed modules under ${path}"
 
-	# TODO?: find way for sane use with dracut (its 90kernel-modules-extra
-	# parses depmod.d files directly and assumes should include its modules
-	# which can lead to unnecessarily increased size or stale modules)
-#	_modules_process_depmod.d "${mods[@]#"${path}/"}"
+	# Instruct Dracut on whether or not we want these modules in initramfs
+	insinto /etc/dracut.conf.d
```

Comment:
insinto+newins should be in a subshell to avoid leaking insinto changes. See `_modules_process_depmod.d` which did something similar,  as a bonus you also get to skip `local` lines.

It's rare but someone could:
```
instinto /dir
...
modules_post_process
doins thinking it's still the same insinto
```

Reply:
Also isn't `dracutconfdir` (`/usr/lib/dracut/dracut.conf.d`) a better place for this?

Reply:
Yes, you are correct! I hadn't realized that dracut also read config files from `/usr/lib/`

Reply:
Fixed, this should also resolve the problem of potentially overriding pre-exisiting user configuration, since whatever new stuff we put in `/usr/lib` gets overridden by `/etc/`.